client/src/components/TaskCard.js

```diff
@@ -19,15 +19,14 @@ const TaskCard = ({ index, task, props, columnId }) => {
     <Draggable draggableId={task._id} index={index}>
       {(provided, snapshot) => (
         <Card
-          className={snapshot.isdragging ? classes.dragged : classes.root}
+          className={snapshot.isDragging ? classes.dragged : classes.root}
           {...provided.draggableProps}
```

Comment:
<img width="509" alt="Screen Shot 2020-03-02 at 2 19 56 PM" src="https://user-images.githubusercontent.com/37638884/75709624-09a45780-5c91-11ea-9e8b-e8d1d8993990.png">
 

this should be lower case like it was

Reply:
thanks. fixed.

Reply:
I removed is dragging.